aten/src/ATen/functorch/BatchRulesRandomness.cpp

```diff
@@ -18,10 +18,10 @@ namespace at {
 namespace functorch {
 
 template <typename F, F Func, typename... ExtraArgs>
-Tensor random_batching_rule(IntArrayRef shape, ExtraArgs... extra_args) {
+Tensor random_batching_rule(SymIntArrayRef shape, ExtraArgs... extra_args) {
   c10::impl::ExcludeDispatchKeyGuard guard(DispatchKey::FuncTorchVmapMode);
   auto maybe_layer = maybeCurrentDynamicLayer();
-  VmapDimVector shapeVec(1, maybe_layer->batchSize());
+  c10::SmallVector<SymInt> shapeVec(1, maybe_layer->batchSize());
```

Comment:
Not sure if this is ok given @ezyang 's latest ambitions?

Reply:
Use SymDimVector instead, but this is fine, I gave up on changing SymIntArrayRef